app/src/main/java/com/example/paetz/yacguide/utils/WidgetUtils.java

```diff
@@ -78,4 +83,33 @@ private static TextView _createTextView(Context context, String text, int textSi
 
         return textView;
     }
+
+    public static String getAscendSymbols(Ascend[] ascends) {
+        Set<String> symbols = new HashSet<>();
+
+        for (Ascend ascend: ascends) {
+            symbols.add(getAscendSymbol(ascend.getStyleId()));
+        }
+
+        StringBuilder result = new StringBuilder();
+        for (String symbol: symbols) {
+            result.append(symbol);
+        }
+        return result.toString();
+    }
+
+    public static String getAscendSymbol(int styleId) {
+        String symbol = " ";
+
+        if (styleId < AscendTypes.BAILED) {
```

Comment:
To prevent this cost-intense if-else-branching for every route's ascend, symbol should be stored in a dedicated class AscendType together with id and name. --> Done in follow-up ticket #52.